src/test/java/io/supertokens/test/WebserverTest.java

```diff
@@ -71,6 +73,90 @@ public void beforeEach() {
+
+            // post request
+            response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendJsonPOSTRequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new JsonObject(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_1);
+            assertEquals("post request from Recipe1", response);
+
+            // put request
+            response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendJsonPUTRequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new JsonObject(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_1);
+            assertEquals("put request from Recipe1", response);
+
+            // delete request
+            response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendJsonDELETERequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new JsonObject(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_1);
+            assertEquals("delete request from Recipe1", response);
+
+        }
+
+        // recipe2 requests
+        {
+            String response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendGETRequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new HashMap<>(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_2);
+            assertEquals("get request from Recipe2", response);
+
+            response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendJsonPOSTRequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new JsonObject(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_2);
+            assertEquals("post request from Recipe2", response);
+
+            response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendJsonPUTRequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new JsonObject(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_2);
+            assertEquals("put request from Recipe2", response);
+
+            response = io.supertokens.test.httpRequest.HttpRequest
+                    .sendJsonDELETERequest(process.getProcess(), "",
+                            "http://localhost:3567/testRecipe", new JsonObject(), 1000,
+                            1000,
+                            null, Utils.getCdiVersion2_7ForTests(), recipe_2);
+            assertEquals("delete request from Recipe2", response);
```

Comment:
Add for when there is no rid given and `recipe_1` should be used

Reply:
alright